Sources/BraveWallet/Crypto/NFT/NFTView.swift

```diff
@@ -108,27 +109,6 @@ struct NFTView: View {
     AssetButton(braveSystemName: "leo.filter.settings", action: {
       isPresentingFiltersDisplaySettings = true
     })
-    .sheet(isPresented: $isPresentingFiltersDisplaySettings) {
-      FiltersDisplaySettingsView(
-        filters: nftStore.filters,
-        isNFTFilters: true,
-        networkStore: networkStore,
-        save: { filters in
-          nftStore.saveFilters(filters)
-        }
-      )
-      .osAvailabilityModifiers({ view in
-        if #available(iOS 16, *) {
-          view
-            .presentationDetents([
-              .fraction(0.6),
-              .large
-            ])
-        } else {
-          view
-        }
-      })
-    }
   }
   
   private var nftHeaderView: some View {
```

Comment:
Can we swap the + and filters button? It would put filters & far right for both Portfolio & NFT